homeassistant/components/telegram_bot/webhooks.py

```diff
@@ -21,7 +21,7 @@
 from homeassistant.components.http.util import get_real_ip
 
 DEPENDENCIES = ['http']
-REQUIREMENTS = ['python-telegram-bot==5.3.0']
+REQUIREMENTS = ['python-telegram-bot==5.3.1']
```

Comment:
If both platforms have the same dependency, let's remove it from platforms and move it to the component.